src/staging/src/styles/search.scss

```diff
@@ -10,6 +10,15 @@
   font-family: $globalFont;
 }
 
+.MuiLink-underlineHover {
+  &:hover {
+    color: $colorBlue;
+    &:focus {
+      color: inherit;
+    }
+  }
+}
+
```

Comment:
I think that this ticket open up a larger issue that we should fix now. There are a ton of different blues on this page, and the anchor and hover colours are all over the place. There is a ton of CSS to handle these colours, and override them, and overriding the override isn't the best solution.

I'm suggesting you should increase the scope of the ticket to normalize the anchor colours on this page, but also site-wide.

I'll add some info to the ticket.